artisoptions_christinenonthermal.h

```diff
@@ -89,6 +89,18 @@ constexpr bool USE_LUT_PHOTOION = false;
 
 constexpr bool USE_LUT_BFHEATING = false;
 
+constexpr bool STRICT_POPULATION_CHECKING = false;
+
+constexpr bool NLTE_LIMIT_ION_STAGES_AFTER_FAILURE = false;
+
+constexpr float STRICT_POPULATION_CHECKING_INVERSION_FACTOR_SOLVER_FAIL = 1000.;
+
+constexpr float STRICT_POPULATION_CHECKING_INVERSION_FACTOR_PRINTOUT_WARNING = 10.;
+
+constexpr double LEVEL_POPULATION_RATIO_WITH_ELEMENT_ALLOW_REMOVE_ION = 1e-9;
```

Comment:
Does this do anything if NLTE_LIMIT_ION_STAGES_AFTER_FAILURE is false? if not, make it clear, e.g.,

NLTE_LIMIT_ION_STAGES_MAX_LEVELPOP_OVER_ELEMENTPOP_REMOVE_ION